environment.yml

```diff
@@ -25,14 +25,14 @@ dependencies:
   - pip
   - pre-commit
   - pydantic
-  - pyproj>=3
-  - rasterio <=1.2.6
+  - pyproj >=3.0
+  - rasterio
   - requests
   - rioxarray
   - scikit-learn ==0.24.2
   - scipy
   - setuptools >50.0
-  - shapely <=1.7.1
+  - shapely
```

Comment:
Oh @Zeitsperre , you'll release another RavenPy to conda? !

Reply:
That;'s the idea. I wanted to ensure that the changes I've already pushed work currently. I can tag and publish today if everything is working.